Source/ProgressionSystemRuntime/Public/Widgets/PSMenuWidget.h

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#pragma once
+
+#include "Blueprint/UserWidget.h"
+#include "PoolManagerTypes.h"
+#include "PSMenuWidget.generated.h"
+
+/**
+ * Widget to display the progression as stars in the main menu.
+ */
+UCLASS()
+class PROGRESSIONSYSTEMRUNTIME_API UPSMenuWidget : public UUserWidget
+{
+	GENERATED_BODY()
+
+public:
+	/**
+	 * Dynamically populates a Horizontal Box with images representing unlocked and locked progression icons.
+	 * @param AmountOfUnlockedPoints The number of images (unlocked-icon as images) to be displayed 
+	 * @param AmountOfLockedPoints The number of images (locked-icon as images) to be displayed
+	 */
+	UFUNCTION(BlueprintCallable, Category= "C++")
+	void AddImagesToHorizontalBox(float AmountOfUnlockedPoints, float AmountOfLockedPoints);
+
+	/**
+	 * Dynamically populates a Horizontal Box with images representing unlocked and locked progression icons.
+	* @param CreatedObjects - Handles of objects from Pool Manager
+	* @param AmountOfUnlockedPoints The number of images (unlocked-icon as images) to be displayed 
+	 * @param AmountOfLockedPoints The number of images (locked-icon as images) to be displayed
+	 */
+	UFUNCTION(BlueprintCallable, Category= "C++")
+	void OnTakeFromPoolCompleted(const TArray<FPoolObjectData>& CreatedObjects, float AmountOfUnlockedPoints, float AmountOfLockedPoints);
```

Comment:
`OnTakeFromPoolCompleted` now is under `public` specified. Please move it below to the `Protected functions`

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/1/commits/1d2c47abf8b9e10f9e36ab5cbf7d4e4a007efaaf